.eslintrc.js

```diff
@@ -1,25 +1,27 @@
-const path = require('path');
-
+const { readFileSync } = require('node:fs');
+const path = require('node:path');
+const ts = require('typescript');
 const { version: reactVersion } = require('react/package.json');
 
+const tsconfigPath = ts.findConfigFile('./', ts.sys.fileExists);
+const { config } = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
+const tsconfig = ts.parseJsonConfigFileContent(config, ts.sys, './');
+
+/**
+ * @type {import('eslint').Linter.Config }
+ */
 module.exports = {
   root: true,
   // Suggested addition from the storybook 6.5 update
   extends: ['plugin:storybook/recommended'],
   // Ignore files which are also in .prettierignore
-  ignorePatterns: [
-    'app/vendor/**',
-    'builds/**/*',
-    'development/chromereload.js',
-    'development/charts/**',
-    'development/ts-migration-dashboard/build/**',
-    'dist/**/*',
-    'node_modules/**/*',
-    'storybook-build/**/*',
-    'jest-coverage/**/*',
-    'coverage/**/*',
-    'public/**/*',
-  ],
+  ignorePatterns: readFileSync('.prettierignore', 'utf8').trim().split('\n'),
```

Comment:
I merged our eslint with the prettier ignore, as the comment here seems to indicate that the ignore lists should match.